examples/cis2-nft/src/lib.rs

```diff
@@ -376,7 +376,7 @@ type TransferParameter = TransferParams<ContractTokenId>;
 /// - Any of the messages sent to contracts receiving a transfer choose to
```

Comment:
Consider rephrasing this doc-comment to remove "messages" and "actions" as that terminology belongs to v0 contracts.